set.mm

```diff
@@ -7069,8 +7069,9 @@ _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
 
   $( An implication and its reverse are equivalent only when both operands are
      equivalent.  The right hand side resembles that of ~ dfbi2 , but ` <-> `
-     is a weaker operator than ` /\ ` .  (Contributed by Wolf Lammen,
-     18-Dec-2023.) $)
+     is a weaker operator than ` /\ ` .  Note that an implication and its
+     reverse can never be simultaniously false, because of ~ pm2.521 .
```

Comment:
```suggestion
     reverse can never be simultaneously false, because of ~ pm2.521 .
```
also, I would change the "only when" to "exactly when", three lines above.

Reply:
done